sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -31,17 +31,17 @@ class ArtifactManifestProperties(object):  # pylint: disable=too-many-instance-a
     :ivar bool can_read: Read Permissions for an artifact.
     :ivar bool can_list: List Permissions for an artifact.
     :ivar architecture: CPU Architecture of an artifact.
-    :vartype architecture: ~azure.containerregistry.ArtifactArchitecture
+    :vartype architecture: Optional[~azure.containerregistry.ArtifactArchitecture]
```

Comment:
What did you reference to determine that all these properties were `Optional`?

Reply:
From line 48.